Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM golang:1-alpine as build
+
+# Build binary
+COPY . /atumd
+WORKDIR /atumd
+RUN go build -a -o "/bin/atumd" .
```

Comment:
I think you should make a static build to ensure that we don't miss any libc elements.

https://github.com/privacybydesign/irmago/blob/02e8186ff9b08afa2151becee2cb7044425be0d7/Dockerfile#L9